models/cow_protocol/ethereum/cow_protocol_ethereum_referrals.sql

```diff
@@ -0,0 +1,38 @@
+-- PoC Query: https://dune.com/queries/1789628?d=1
+WITH
+referral_map as (
+    select
+        distinct app_hash,
+        referrer
+    from {{ ref('cow_protocol_ethereum_app_data') }}
+    where referrer is not null
+)
+
+-- Table with first trade per user. Used to determine their referral
+,ordered_user_trades AS (
```

Comment:
Nitpick: Not a fan of having commas before the name. Could also just be my personal style.

```suggestion
),

-- Table with first trade per user. Used to determine their referral
ordered_user_trades AS (
```

Reply:
It is also a preferred style for me, but I have recently become more interested in the Left comma club... (easier to comment out blocks). I think I'll leave it but, won't make it a habit if thats ok with you.

Reply:
I think @jeff-dude would be proud I am leaning toward the left-comma-club.